src/components/bigLogo.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import InViewMonitor from 'react-inview-monitor';
+import MtSvgLines from 'react-mt-svg-lines';
+
+import '../../node_modules/animate.css/animate.min.css';
+
+import LogoOnly from '../images/logoOnly.svg';
+import LogoVandyHacksText from '../images/textVandyHacks.svg';
+import Grid from '../images/gridLogo.svg';
+import LocationDateText from '../images/locationDateText.svg';
+
+const Container = styled.div`
+	width: 76%;
```

Comment:
76 is a weird # lol

Reply:
lol true these styling should be gerneralized later QAQ